src/main/java/seedu/cafectrl/data/CurrentDate.java

```diff
@@ -3,24 +3,26 @@
 import java.util.ArrayList;
 
 public class CurrentDate {
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
```

Comment:
perhaps you could name these variables with something more meaningful such as:
```suggestion
    private static final int MIN_ORDER_LIST_SIZE = 0;
    private static final int DAY_BASE_NUMBER = 0;
    private static final int DAY_OFFSET = 1;
    private static final int ORDER_LIST_SIZE_OFFSET = 1;
```